server/cmwell-ws/app/controllers/Application.scala

```diff
@@ -442,18 +442,19 @@ callback=< [URL] >
       case Failure(e) => asyncErrorHandler(e)
       case Success(request) => {
         val normalizedPath = normalizePath(request.path)
+        val isPriorityWrite = originalRequest.getQueryString("priority").isDefined
```

Comment:
How `.flatMap(asBoolean).getOrElse(false)` is more concise than `.isDefined` ?
Anyway, "false" won't be a use case. We are going to parse this value as an Integer in phase 2.
For this phase I'm documenting the API to use this query parameter without value, so I hope users won't do `?priority=false` just like they don't do `?with-data=false` because it is simply redundant. Nonetheless, I understand why you wanted to use `asBoolan`, and I would've done it if I wasn't keeping in mind it will soon be an Int. What do you think?